tests/unit/type/set.tcl

```diff
@@ -33,6 +33,7 @@ start_server {
         assert_equal {0 1} [r smismember myset bla foo]
         assert_equal {0} [r smismember myset bla]
         assert_equal "bar $initelems($type)" [lsort [r smembers myset]]
+        r memory usage myset
```

Comment:
Added for debugging?

Reply:
To ensure the command runs without crashing. idk, what do you think? 🤷🏻‍♀️

Reply:
To increase coverage? Ok, yes, good.